### DISCUSSION TITLE
GitHub Education coupon expiration reminder

### DISCUSSION THREAD

**Discussion by HugoGresse** [community/community#44649] [GitHub Education]
Hi,

A while ago, a teacher in the same university registered the organisation in GitHub Education.

Two weeks ago, I received an email from GH saying the coupon will expire in 14d. I then registered using my university email, was verified and my account is now a "pro" account.
This week, while being already verified, I've received again the email saying the coupon will still expire (on a different university email though). 

How can I check that the **GH organisation** will still benefit from GitHub Education?

Organisation: https://github.com/plantnet

Thanks for the support

**Comment by ahsanur** [comment]
Same issue here.
